packages/react-devtools-shared/src/__tests__/utils-test.js

```diff
@@ -123,51 +124,51 @@ describe('utils', () => {
     });
   });
 
-  describe('format', () => {
-    // @reactVersion >= 16.0
+  describe('formatConsoleArgumentsToSingleString', () => {
     it('should format simple strings', () => {
-      expect(format('a', 'b', 'c')).toEqual('a b c');
+      expect(formatConsoleArgumentsToSingleString('a', 'b', 'c')).toEqual(
+        'a b c',
+      );
     });
 
-    // @reactVersion >= 16.0
```

Comment:
We don't really need this pragma here, because `format` has nothing to do with different versions of React, so there is no reason to test it against them.